ci/format_pre.sh

```diff
@@ -57,14 +57,8 @@ if [[ -n "$AZP_BRANCH" ]]; then
     "${ENVOY_SRCDIR}"/tools/code_format/check_format_test_helper.sh --log=WARN
 fi
 
-fix_format () {
-    echo "Fixing format..."
-    "${ENVOY_SRCDIR}"/tools/code_format/check_format.py fix
-    return 1
-}
-
 CURRENT=check_format
-"${ENVOY_SRCDIR}"/tools/code_format/check_format.py check || fix_format
+"${ENVOY_SRCDIR}"/tools/code_format/check_format.py fix
```

Comment:
Don't you need to specify fail on diff here? Can you make a format error and double check that CI fails? Thank you.

/wait-any

Reply:
good spot, apologies, fixed